### PR TITLE
fix: Upgrade to skaffold/v3

### PR DESCRIPTION
**Pull request by mmcloud**
### Background 

Skaffold has released a major new update.
### Change Summary

Run `skaffold fix` with new skaffold v2.0.0 binarys. 

### Additional Notes

For upgrade notes see:
https://skaffold.dev/docs/upgrading/

### CODE REVIEW

**Comment by mmcloud** [comment]
A new runner with the skaffold V2 binary requires the runner to be rebuilt with the updated install-dependencies.sh see - https://github.com/GoogleCloudPlatform/microservices-demo/tree/main/.github/workflows#appendix---creating-a-new-actions-runner.

**Comment by NimJay** [comment]
First off, thank you, @mmcloud, for creating this pull-request! :)

Do we also need to update https://github.com/mmcloud/microservices-demo/blob/skaffold-v2/docs/development-guide.md#prerequisites?

**Comment by NimJay** [comment]
I just upgrade the `skaffold` version (to v2.0.1) on the 2 virtual machines (GitHub Action runners) used by this repository for CI.
I merely followed the [instructions here](https://github.com/GoogleCloudPlatform/bank-of-anthos/pull/514#issuecomment-883542251).

File: .github/workflows/install-dependencies.sh [hunk 1]
```diff
@@ -56,7 +56,7 @@ sudo ln -s $HOME/go/bin/addlicense /bin
 sudo apt install -y build-essential
 
 # install skaffold
-curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/v2.0.0/skaffold-linux-amd64 && \
+curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/latest/skaffold-linux-amd64 && \
```
Comment (NimJay) [inline .github/workflows/install-dependencies.sh:59]: Praise: Good call. Let's stick to `latest`.

Comment (mmcloud) [reply .github/workflows/install-dependencies.sh:59]: thank you for checking the latest binaries!

**Review by NimJay — APPROVED** [review]
Thanks again for doing this, @mmcloud!
All checks are passing, and the staging URL (from this pull-request) work.
Approved!

**Pull request merged**